setup.py

```diff
@@ -56,5 +56,6 @@ def run_tests(self):
     """,
     zip_safe=False,
     install_requires=["decorator>=4.0.0"],
+    tests_require=["decorator>=4.0.0"],
```

Comment:
iiuc this option only applies to the "python setup.py test" command which we dont support anymore as its deprecated.

Reply:
I just tested `tox` (which was failing) without this, and the tests run correctly.

shall i remove it?

Reply:
I think so because they want to blow away "python setup.py test" entirely, which means this setting also goes away.  unless there's some other way it's consumed I'm not aware of.

Reply:
ok. removed.